README.md

```diff
@@ -25,6 +25,7 @@ Here are the steps to get the host environment ready for this image:
 	```bash
 	docker run -it --privileged --net=host --restart=always \
 	-e PI_RESET_PIN=<optional-see-explanation-below> \
+	-e GATEWAY_REGION=<optional-see-explanation-below> \
 	-e GATEWAY_EUI=RASPBERRY_PI_MAC_ADDRESS \
 	-e GATEWAY_LAT=YOUR_LATITUDE \
```

Comment:
More as a side note: I was wondering if the LAT/LON actually have an impact for TTN since they are also configurable on the TTN interface